iOS-Email-Client/AsyncTasks/SendMailAsyncTask.swift

```diff
@@ -212,14 +212,26 @@ class SendMailAsyncTask {
             guard let myAccount = SharedDB.getAccountByUsername(self.username) else {
                 return
             }
-            guard case let .SuccessArray(keysArray) = responseData else {
+            guard case let .SuccessDictionary(keysArray) = responseData else {
                 self.setEmailAsFailed()
                 DispatchQueue.main.async {
                     completion(responseData)
                 }
                 return
             }
-            for keys in keysArray {
+            let keyBundles = keysArray["keyBundles"] as! [[String:Any]]
+            let blackListedDevices = keysArray["blacklistedKnownDevices"] as! [[String:Any]]
+            let store: CriptextSessionStore = CriptextSessionStore()
```

Comment:
Don't use CriptextSessionStore. Use `DBAxolotl`

Reply:
The class CriptextSessionStore has a method deleteSession which uses DBAxolotl deleteSessionRecord